src/main/java/com/google/firebase/auth/UserRecord.java

```diff
@@ -73,6 +79,17 @@
       }
     }
     this.userMetadata = new UserMetadata(response.getCreatedAt(), response.getLastLoginAt());
+    String customClaims = response.getCustomClaims();
+    if (!Strings.isNullOrEmpty(customClaims)) {
+      Map<String, Object> parsed = new JSONObject(customClaims).toMap();
+      if (parsed.isEmpty()) {
+        this.customClaims = null;
+      } else {
+        this.customClaims = ImmutableMap.copyOf(parsed);
```

Comment:
Is it possible to delete a single custom claim?

Reply:
Something like:

```
Map<String, Object> copy = new HashMap(userRecord.getCustomClaims());
copy.remove("someClaim");
updateRequest.setCustomClaims(copy);
```